pkgs/top-level/python-packages.nix

```diff
@@ -8247,6 +8247,21 @@ in {
     inherit (pkgs) which;
   };
 
+  nixpkgs = if pythonAtLeast "3.5" then null else buildPythonPackage rec {
```

Comment:
It's better to set the `disabled` attribute on the derivation than returning `null`.
I think this logic is inverted, currently this package is only available for python2.

I think what you actually want is
`disabled = ! pythonAtLeast "3.5"`.

Reply:
Changed to `disabled = ! pythonAtLeast "3.5"`